### PR TITLE
Feat/production template support

### PR DESCRIPTION
**Pull request by aragon999**

### CODE REVIEW

File: extension/asset.go [hunk 1]
```diff
@@ -47,70 +47,103 @@ func BuildAssetsForExtensions(shopwareRoot string, extensions []Extension) error
...
 					return err
 				}
 			}
 		}
 
-		npmInstallCmd := exec.Command("npm", "--prefix", fmt.Sprintf("%s/src/Administration/Resources/app/administration/", shopwareRoot), "install") //nolint:gosec
-		npmInstallCmd.Stdout = os.Stdout
-		npmInstallCmd.Stderr = os.Stderr
-		err := npmInstallCmd.Run()
+		if _, err := os.Stat(fmt.Sprintf("%s/vendor/shopware/administration/Resources/app/administration/package.json", shopwareRoot)); err == nil {
```
Comment (shyim) [inline extension/asset.go:58]: Can we build this similar to https://github.com/shyim/shopware-docker/blob/v1/functions.sh#L183-L193 ?

Comment (shyim) [inline extension/asset.go]: Can we build this similar to https://github.com/shyim/shopware-docker/blob/v1/functions.sh#L183-L193 ?

**Review by shyim — CHANGES_REQUESTED** [review]

File: extension/asset.go (hunk 1 above)
Comment (aragon999) [reply extension/asset.go]: Yes we probably should also consider it here: https://github.com/FriendsOfShopware/shopware-cli/blob/main/cmd/project/platform.go#L54-L62

**Comment by shyim** [comment]
@aragon999 can you solve the conflict?

**Comment by aragon999** [comment]
@shyim ah yes done, sorry missed that :-)

**Pull request merged**